app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
@hweng I agree with Matt. I think it may not be necessary to hardcode for the jpeg mime type here. The major issue here is that some of the .jpg derivatives generated in the past may be corrupted overtime for some reason, and we need to find out those corrupted .jpg derivative and fixed them, though sometimes it may be still displayable.